21CB46/HTML-CSS
# ADD- An Adding The HTML-CSS Project.

I am adding the HTML-CSS project which is the Parallax Website.

ADD #4 

Video-

https://github.com/21CB46/HTML-CSS/assets/91898349/4d9c8cd1-190b-43df-886f-d86a0ad37134

How should this be tested?
1. Go to the IDE..
2. Open the parallax website folder.
3.  and see the code.
